lib/pages/homeserver_picker/homeserver_picker_view.dart

```diff
@@ -148,79 +152,107 @@ class HomeserverTextField extends StatelessWidget {
 
   @override
   Widget build(BuildContext context) {
-    return TypeAheadField(
-      textFieldConfiguration: TextFieldConfiguration(
-        autofocus: controller.state != HomeserverState.ssoLoginServer,
-        autocorrect: false,
-        enabled: true,
-        controller: controller.homeserverController,
-        decoration: InputDecoration(
-          border: OutlineInputBorder(
-            borderSide:
-                BorderSide(color: Theme.of(context).colorScheme.outline),
-          ),
-          focusedBorder: controller.state == HomeserverState.wrongServerName
-              ? OutlineInputBorder(
-                  borderSide: BorderSide(
-                    color: Theme.of(context).colorScheme.error,
-                    width: 2.0,
-                  ),
-                )
-              : null,
-          labelText: controller.state != HomeserverState.wrongServerName
-              ? L10n.of(context)!.homeserver
-              : L10n.of(context)!.wrongServerName,
-          labelStyle: Theme.of(context).textTheme.bodySmall?.copyWith(
-                letterSpacing: 0.4,
-                color: controller.state != HomeserverState.wrongServerName
-                    ? Theme.of(context).colorScheme.onSurface
-                    : Theme.of(context).colorScheme.error,
+    final useShortCuts = (PlatformInfos.isWeb ||
+        PlatformInfos.isDesktop ||
+        AppConfig.sendOnEnter);
+
+    return Shortcuts(
+      shortcuts: useShortCuts
+          ? <LogicalKeySet, Intent>{
```

Comment:
Mobile and tablet platforms don't need Actions widgets. So, why don't you create another widget called `DesktopHomeserverTextField` then ....

Reply:
You are right, tell me if it's good for you now